app/javascript/legacy_react/src/lib/utils.ts

```diff
@@ -15,3 +15,7 @@ export function castToUndefinedIfBlank(i:null|undefined|string) :
     string | undefined {
   return isBlank(i) ? undefined : i;
 }
+
+export function removeChar(i:null|undefined|string, charArray:null|undefined|string) {
```

Comment:
Just noticed that there isn't a return type for this function. Make sure to add that.